oozie/src/main/java/org/apache/falcon/oozie/OozieBundleBuilder.java

```diff
@@ -106,11 +108,24 @@ public Properties build(Cluster cluster, Path buildPath) throws FalconException
         properties.setProperty(AbstractWorkflowEngine.NAME_NODE, ClusterHelper.getStorageUrl(cluster));
 
         //Add libpath
-        String libPath = getLibPath(buildPath);
-        if (libPath != null) {
-            properties.put(OozieClient.LIBPATH, getStoragePath(libPath));
+        String path = getLibPath(buildPath);
+        if (StringUtils.isNotBlank(path)) {
+            String storageLibPaths = null;
+            String[] libPaths = path.split(WF_LIB_SEPARATOR);
+            for (String libPath : libPaths) {
```

Comment:
I removed isNotBlank check to add WF_LIB_SEPARATOR. 

Now code is 
storageLibPaths = StringUtils.isNotBlank(storageLibPaths) ?
                            storageLibPaths.concat(EntityUtil.WF_LIB_SEPARATOR).concat(tempPath) : tempPath;

This is required to avoid NPE and also separator should be avoided if there are multiple entries.